AllReadyApp/Web-App/AllReady/Services/TaskAttachmentService.cs

```diff
@@ -33,13 +33,13 @@ public AttachmentService(IOptions<AzureStorageSettings> options)
         /// <param name="organizationId">int ID</param>
         /// <param name="attachment">a attachment from Microsoft.AspNet.Http</param>
         /// <returns>URL to the uploaded file</returns>
-        public async Task<string> UploadTaskAttachmentAsync(int taskId, IFormFile attachment)
+        public async Task<string> UploadAsync(int taskId, IFormFile attachment)
```

Comment:
please update the xml comments with what's actually being passed to `UploadAsync`.